deploy/rbd/kubernetes/v1.14+/csi-rbdplugin.yaml

```diff
@@ -72,6 +72,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: POD_NAMESPACE
```

Comment:
@sbskas if I remember correctly this is required if  you are doing some operation with  PVC created with  1.0.0 ceph-csi

Reply:
I think this is required if you deploy in another namespace also.

Reply:
what you said is correct and as per https://github.com/ceph/ceph-csi/blob/46b19a93f5485ad5bb248fd638cfe8c8da220efa/pkg/util/k8scmcache.go#L54 required to support operate on PVC created using ceph-csi v1.0.0